docs/testing/debug-add-ins-using-f12-developer-tools-on-windows-10.md

```diff
@@ -1,30 +1,43 @@
 ---
-title: Debug add-ins using F12 developer tools on Windows 10
+title: Debug add-ins using developer tools on Windows 10
 description: ''
-ms.date: 03/19/2019
+ms.date: 07/01/2019
 localization_priority: Priority
 ---
 
-# Debug add-ins using F12 developer tools on Windows 10
+# Debug add-ins using developer tools on Windows 10
+
+If you are not using an IDE like Visual Studio, or if you need to investigate a problem while running your add-in outside the IDE, there are developer tools available to help you debug the add-in on Windows 10. 
+
+The tool that you use depends on whether the add-in is running in Edge or Internet Explorer. This is determined by the version of Windows 10 and the version of Office that are installed on the computer. For more information about determining which browser is being used on your development computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). 
```

Comment:
Second sentence: Would it help to specify this is for win32? Like "For Office on Windows, this is determined..."

Reply:
Not really. The dev is debugging JS, which should run the same on all compliant browsers, so they aren't debugging "for Office on X" for any X.  They are just debugging … period. What matters is that they are doing their JS debugging on a Windows computer.